mantle/kola/tests/ignition/mount.go

```diff
@@ -54,7 +54,7 @@ func testMountDisks(c cluster.TestCluster) {
 	config := setupIgnitionConfig()
 
 	options := platform.MachineOptions{
-		AdditionalDisks: []string{"1024M", "1024M"},
+		AdditionalDisks: []string{"1G", "1G"},
```

Comment:
Hmm tricky...if we were supporting the `M` specifier before it may be safer to continue doing so.

It looks like all of the external tests use `G` though.

Reply:
That would be better, but we intend to get size as int (trim sufix `G`) in the` ParseDiskSpec func`:
- for qemu, need to re-add `G` as it needs string to be passed to [qemu-img](https://github.com/coreos/coreos-assembler/blob/main/mantle/platform/qemu.go#L1049-#L1052)
- for gcp and other cloud, need int, as the default is size in `G`

WDYT?

Reply:
Yeah, it seems a bit hard to do without re-implementing the suffix parsing here entirely which would be unfortunate.

I think this is OK as is!